fi/development.html

```diff
@@ -0,0 +1,210 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+    <title>Kehitys</title>
+    <link href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css" rel="stylesheet" type="text/css" />
+    <!-- <link rel="stylesheet" href="/static/css/base.css" type="text/css" media="screen" /> -->
+    <!-- <link rel="stylesheet" id="current-theme" href="/static/css/style.css" type="text/css" media="screen" /> -->
+    <link rel="stylesheet" href="/static/external/css/reset.css" type="text/css" media="screen" />
+    <link rel="stylesheet" href="/static/external/css/responsive-gs-12col.css" type="text/css" media="screen" />
+    <link rel="stylesheet" href="/static/css/base.css" type="text/css" media="screen" />
+    <link href="http://fonts.googleapis.com/css?family=Open+Sans:300,400,700|Gentium+Basic" rel="stylesheet" type="text/css" />
+    <link rel="icon" href="/static/SymPy-Favicon.ico" />
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.2/jquery.min.js"></script>
+    <script type="text/javascript" src="/static/javascript/main.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <h1><a href="index.html"> <img src="/static/images/logo.png" alt="SymPy Logo" /> SymPy</a></h1>
+        <div id="mobile-menu"><i class="icon-ellipsis-vertical"></i></div>
+      </header>
+      <nav id="main-navigation" class="row">
+        
+        
+        <ul>
+          <li class=""><a href="index.html">Pääsivu</a></li>
+          <li class=""><a href="features.html">Ominaisuudet</a></li>
+          <li class=""><a href="download.html">Lataa</a></li>
+          <li class=""><a href="http://docs.sympy.org">Dokumentaatio <i class="icon-external-link-sign"></i></a></li>
+          <li class=""><a href="support.html">Tuki</a></li>
+          <li class="active"><a href="development.html">Kehitys</a></li>
+          <li class=""><a href="donate.html">Lahjoita</a></li>
+          <li><a href="http://live.sympy.org/">Online-komentokehoite <i class="icon-external-link-sign"></i></a></li>
+        </ul>
+        
+      </nav>
+      <section class="row" id="main">
+        <div class="col span_7 clr small_span_4">
+          
+          <article id="content">
+            
+
+<h2 class="title">Kehitys</h2>
+<div class="inner">
+  <p class="first">
+    Me olemme kiinnostuneita mistä tahansa avustuksesta. On olemassa monta tapaa jolla voit
+      avustaa:
+    <ul>
+      <li>
+        Mikäli kirjoitit jotakin kiinnostavaa SymPy-ohjelmistolla, ole hyvä ja lähetä meille patch
```

Comment:
I would very much like to see 'patch' translated. For example, 'korjaustiedosto' (or sometimes just 'korjaus').